internal/network/manifests_generator.go

```diff
@@ -81,10 +81,19 @@ spec:
         overwrite: true
 `
 
-const snoDnsmasqConf = `
-address=/apps.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/HOST_IP
-address=/api-int.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/HOST_IP
-address=/api.{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}/HOST_IP
+const snoDnsmasqConf = `#!/usr/bin/env bash
+
+CLUSTER_FULL_DOMAIN="{{.CLUSTER_NAME}}.{{.DNS_DOMAIN}}"
+if [ -f /etc/default/cluster-domain ]; then
+    CLUSTER_FULL_DOMAIN=$(cat /etc/default/cluster-domain)
+fi
+HOST_IP=$(cat /var/run/nodeip-configuration/primary-ip)
```

Comment:
it should always be there?

Reply:
Yep, nodeip configuration is always running and creating this file

Reply:
In all versions?

Reply:
Should we wait for the file to be created, rather than assuming it exists?